### ISSUE TITLE
Your watchnixtoon not working

### ISSUE THREAD

**Issue by daniel-ntl** [nixgates/Incursion#47]
Hi sir, the last 3 days from this day 8 april 2018 your watchnixtoon addon notworking but the website is ok, please help, thx sir

**Comment by fpanico** [comment]
Also not working for me

**Comment by major48** [comment]
Yep not working for me also. I made a GitHub account just to contact you. Great addon btw

**Comment by bjgood** [comment]
Not working for me either. Hope it's fixed soon. The kids miss it.

Thanks, nixgates, for any help on this matter.

Later dude,
Bob

**Comment by dokoab** [comment]
It seems someone fixed it, it's due a revision before the addon is updated automatically.
If you're in a hurry, you can download this file...
https://raw.githubusercontent.com/alexnander/Incursion/b2d367d796c875a5f927efcf445c77cfb0ffc020/plugin.video.watchnixtoons/default.py

...and copy it over the same-name one installed in Kodi. It's in the **plugin.video.watchnixtoons** folder inside the Kodi addons folder:  
**Android:** /sdcard/Android/data/org.xbmc.kodi/files/.kodi/addons/ (the .kodi folder is hidden, use ES File Explorer with the "Show hidden files" option)  
**Windows:** AppData/Roaming/Kodi/addons/  
**Linux:** /home/USER/.kodi/addons/  

Again, you only need to do this if you want the update *immediately*. Otherwise just wait and the addon will be updated automatically some time from now (no idea when).

**Comment by major48** [comment]
Thanks. Works great again

**Comment by daniel-ntl** [comment]
Already tried, i have replaced the default.py with the link but still not working, is it because i still use kodi jarvis 16.1, ??

**Comment by dokoab** [comment]
@daniel-ntl if you're still having trouble even after the update, copy the error parts of your Kodi log (/.kodi/temp/kodi.log) after the addon causes that error and share it here.
The log says what's going on.

**Comment by daniel-ntl** [comment]
yup now i know why it didnt work in jarvis 16.1 its because the phyton.....so now i'm using cemc, and the addons works but i have some issues if you already play 01 episode on the same page, the next one the 02 episode cannot play, how come?
